lib/pbench/test/functional/server/test_connect.py

```diff
@@ -21,6 +21,7 @@ class TestConnect:
         "login",
         "logout",
         "register",
+        "server_configuration",
```

Comment:
At lines 41-44, we check that all existing endpoints are expected; however, we don't confirm that all expected endpoints actually exist.  It strikes me that the latter is more important.  (Also, it occurs to me that we're doing "set operations" here, and that we could do both checks with a `[symmetric_]difference` set operation (or maybe just an equality check).

Reply:
I had a similar thought at one point, but I really don't want to get into reworking the actual *tests* at this point beyond what was necessary to get them working.

I do plan to improve and extend the tests after this is in; this is one of the things I would look at.